packages/components/src/templates/next/components/internal/DatadogRum/DatadogRum.tsx

```diff
@@ -0,0 +1,30 @@
+"use client"
+
+import { datadogRum } from "@datadog/browser-rum"
+
+const PUBLIC_DD_RUM_CLIENT_TOKEN = "pub277c32b9cf49ccee087ec9dc83d62438"
+const PUBLIC_RUM_APPLICATION_ID = "3516693f-2249-4677-b962-5bfa96d5f556"
+
+datadogRum.init({
+  applicationId: PUBLIC_RUM_APPLICATION_ID,
+  clientToken: PUBLIC_DD_RUM_CLIENT_TOKEN,
+  site: "datadoghq.com",
+  service: "isomer-next-sites",
+  env: "production",
+  sessionSampleRate: Number(
+    process.env.NEXT_PUBLIC_ISOMER_NEXT_RUM_SAMPLING_RATE ?? 0,
+  ),
+  sessionReplaySampleRate: Number(
+    process.env.NEXT_PUBLIC_ISOMER_NEXT_RUM_SAMPLING_RATE ?? 0,
+  ),
+  trackUserInteractions: true,
+  trackResources: true,
+  trackLongTasks: true,
+  defaultPrivacyLevel: "allow",
+})
```

Comment:
should we pass in `production` as ENV too? or check that its production/staging before rendering. so as to not dirty the data + not have it run on dev and incur unncessary cost

Reply:
Yup, the env is provided earlier on line 13, but I also added a check to only render this component if the site is not on staging